Source/Login/Web/select-provider/IdentityProviderListItem.tsx

```diff
@@ -25,7 +25,8 @@ export class IdentityProviderListItem extends React.Component<IdentityProviderLi
             <DocumentCard type={DocumentCardType.compact} onClick={this.onClick}>
                 <DocumentCardPreview previewImages={[
                     {
-                        previewImageSrc: 'https://docs.microsoft.com/en-us/azure/active-directory/develop/media/howto-add-branding-in-azure-ad-apps/ms-symbollockup_mssymbol_19.svg',
+                        // previewImageSrc: 'https://docs.microsoft.com/en-us/azure/active-directory/develop/media/howto-add-branding-in-azure-ad-apps/ms-symbollockup_mssymbol_19.svg',
+                        previewImageSrc: this.props.provider.imageURL || 'https://docs.microsoft.com/en-us/azure/active-directory/develop/media/howto-add-branding-in-azure-ad-apps/ms-symbollockup_mssymbol_19.svg',
```

Comment:
Maybe we just don't show an image if the imageURL is not set?

Reply:
So empty string? Or not set the property?

Reply:
Will this fix it? https://github.com/dolittle-platform/Authentication/pull/10/commits/6c603490e50540c7fda28d626abfb64db4152004

Reply:
It worked when I tested it locally

Reply:
Tested it just now myself, it complained about an unknown source. I have another suggestion

Reply:
Weird. Then come with it :P